code/controllers/subsystem/codex.dm

```diff
@@ -1,7 +1,8 @@
 SUBSYSTEM_DEF(codex)
 	name = "Codex"
-	flags = SS_NO_FIRE
+	flags = SS_HIBERNATE | SS_BACKGROUND
 	init_order = INIT_ORDER_CODEX
+	wait = 1
 
```

Comment:
```suggestion
	flags = SS_HIBERNATE | SS_BACKGROUND | SS_TICKER
	init_order = INIT_ORDER_CODEX
	priority = FIRE_PRIORITY_CODEX
	wait = 1
```

Define FIRE_PRIORITY_CODEX in the "background bucket" section of subsystems.dm, with a value of 5